src/controllers/scripting/legacy/controllerscriptinterfacelegacy.cpp

```diff
@@ -298,7 +298,8 @@ void ControllerScriptInterfaceLegacy::triggerScriptConnection(
         return;
     }
 
-    connection.executeCallback(coScript->get());
+        connection.executeCallback(coScript->get());
```

Comment:
Unrelated change. 
There are a few other changes like this.
Maybe it is treasonable to remove them and then squash all commits to one or two.